cosmwasm/packages/wasmi-runtime/src/crypto/secp256k1.rs

```diff
@@ -1,3 +1,62 @@
+use log::*;
+
+use crate::cosmwasm::encoding::Binary;
+use crate::cosmwasm::types::CanonicalAddr;
+use crate::crypto::traits::PubKey;
+use crate::crypto::CryptoError;
+use ripemd160::{Digest, Ripemd160};
+use secp256k1::Secp256k1;
+use sha2::{Digest as Sha2Digest, Sha256};
+
+// TODO: Find a way to implement this better. secp256k1 is not ported to sgx, thus does not implement mesalock's serde and cannot be used
+pub type Secp256k1PubKey = Vec<u8>;
```

Comment:
`secp256k1` in not ported by mesalock - so it uses the rust `std`'s `serde` implementation instead of mesalock's port and thus conflicting with eachother